tracer.go

```diff
@@ -60,6 +60,21 @@ func recordError(span trace.Span, err error) {
 	}
 }
 
+const sqlOperationUnknkown = "UNKNOWN"
+
+// sqlOperationName attempts to get the first 'word' from a given SQL query, which usually
+// is the operation name (e.g. 'SELECT').
+func sqlOperationName(query string) string {
```

Comment:
Could we add a unit test with a couple of examples here?

Reply:
Done